bin/accumulo-testing

```diff
@@ -229,7 +229,7 @@ rw-local)
     exit 1
   fi
   build_shade_jar
-  java -Dlog4j.configuration="file:$log4j_config" -cp "$at_shaded_jar" "$randomwalk_main" "$at_props" "$2"
+  java -Dlog4j.configuration="file:$log4j_config" -cp "$at_shaded_jar:$HADOOP_CONF_DIR" "$randomwalk_main" "$at_props" "$2"
```

Comment:
One way we can decouple this from specific environment variables (which may or may not exist), is to simply remove the `-cp` section, and set `CLASSPATH` above the call to `java`.
Something like `export CLASSPATH="$at_shaded_jar:$CLASSPATH"`

That way, the caller can simply set `CLASSPATH=/path/to/hadoop/confdir` prior to running the `accumulo-testing` command.

We could also add a sanity check prior to running java to make sure that the `$CLASSPATH` contains a directory with `core-site.xml` and/or `hdfs-site.xml` (or whatever the Hadoop client configuration uses by default).